runtime/src/main/java/org/corfudb/runtime/collections/ICorfuMap.java

```diff
@@ -5,7 +5,7 @@
 import java.util.function.Predicate;
 
 public interface ICorfuMap<K, V>
-    extends Map<K, V> {
+    extends StreamingMap<K, V> {
```

Comment:
I think these should be reverse. `ICorfuMap` is the "more general" type.

Reply:
`StreamingMap` is not a simplified interface.